languages/go/exercises/concept/iteration/iteration_test.go

```diff
@@ -0,0 +1,111 @@
+package iteration
+
+import "testing"
+
+func TestFibonacci(t *testing.T) {
+	type args struct {
+		n int
+	}
+	tests := []struct {
+		name string
+		args args
```

Comment:
I think `args` type is unnecessary and we could use `n int` directly. It would also make the test cases more compact. Same applies for the other test.

Reply:
Yes. I will change that.

Note: My IDE auto-creates the tests and I do not always change the structure.